### PR TITLE
Barracuda WAF pack

### PR DESCRIPTION
**Pull request by edik24**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
Due to timing constraints - Demo was scheduled for May 8th.
but the review can start

**Review by ShahafBenYakir — COMMENTED** [review]
This looks ok on my side

File: Packs/BarracudaWAF/README.md
Comment (eepstain) [inline Packs/BarracudaWAF/README.md:1]: @edik24,
We should number the steps and please add @ShirleyDenkberg as an assignee so she could review your README.

Comment (edik24) [reply Packs/BarracudaWAF/README.md:1]: added numbers, and addin @ShirleyDenkberg to to PR

File: Packs/BarracudaWAF/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Barracuda WAF
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+In Barracuda WAF appliance:
+1. Go to the ADVANCED > Export Logs page.
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:5]:
```suggestion
1. Go to the **ADVANCED** > **Export Logs** page. 
```

File: Packs/BarracudaWAF/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Barracuda WAF
+This pack includes Cortex XSIAM content. 
+## Configuration on Server Side
+In Barracuda WAF appliance:
+1. Go to the ADVANCED > Export Logs page. 
+2. In the Export Logs section, click Add Export Log Server. The Add Export Log Server window opens. Specify values for the following::
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:6]:
```suggestion
2. In the Export Logs section, click **Add Export Log Server**. The Add Export Log Server window opens. Specify values for the following::
```

File: Packs/BarracudaWAF/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   - Name – Enter a name for the syslog NG server. 
+   - Log Server Type – Select Syslog NG.
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:8]:
```suggestion
   - Log Server Type – **Select Syslog NG**. 
```

File: Packs/BarracudaWAF/README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+   - Log Server Type – Select Syslog NG. 
+   - IP Address or Hostname – Enter the IP address or the hostname of the syslog NG server. 
+   - Port – Enter the port associated with the IP address of the syslog NG server. 
+   - Connection Type – Select the connection type to transmit the logs from the Barracuda Web Application Firewall to the syslog server. UDP is the default port for syslog communication. UDP, TCP, or SSL can be used in case of NG syslog server. 
+   - Validate Server Certificate – Set to Yes to validate the syslog server certificate using the internal bundle of Certificate Authority's (CAs) certificates packaged with the system. If set to No, any certificate from the syslog server is accepted. 
+   - Client Certificate – When set to Yes, the Barracuda Web Application Firewall presents the certificate while connecting to the syslog server. 
+   - Certificate – Select a certificate for the Barracuda Web Application Firewall to present when connecting to the syslog server. Certificates can be uploaded on the BASIC > Certificates page. For more information on how to upload a certificate, see How to Add an SSL Certificate.
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:14]:
```suggestion
   - Certificate – Select a certificate for the Barracuda Web Application Firewall to present when connecting to the syslog server. Certificates can be uploaded on the **BASIC** > **Certificates** page. For more information on how to upload a certificate, see [How to Add an SSL Certificate](https://campus.barracuda.com/product/webapplicationfirewall/doc/4259930/how-to-add-an-ssl-certificate/).
```

File: Packs/BarracudaWAF/README.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+   - Connection Type – Select the connection type to transmit the logs from the Barracuda Web Application Firewall to the syslog server. UDP is the default port for syslog communication. UDP, TCP, or SSL can be used in case of NG syslog server. 
+   - Validate Server Certificate – Set to Yes to validate the syslog server certificate using the internal bundle of Certificate Authority's (CAs) certificates packaged with the system. If set to No, any certificate from the syslog server is accepted. 
+   - Client Certificate – When set to Yes, the Barracuda Web Application Firewall presents the certificate while connecting to the syslog server. 
+   - Certificate – Select a certificate for the Barracuda Web Application Firewall to present when connecting to the syslog server. Certificates can be uploaded on the BASIC > Certificates page. For more information on how to upload a certificate, see How to Add an SSL Certificate. 
+   - Log Timestamp and Hostname – Set to Yes if you want to log the date and time of the event, and the hostname configured on the BASIC > IP Configuration > Domain Configuration section.
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:15]:
```suggestion
   - Log Timestamp and Hostname – Set to Yes if you want to log the date and time of the event, and the hostname configured in the **BASIC** > **IP Configuration** > **Domain Configuration** section. 
```

File: Packs/BarracudaWAF/README.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+   - Certificate – Select a certificate for the Barracuda Web Application Firewall to present when connecting to the syslog server. Certificates can be uploaded on the BASIC > Certificates page. For more information on how to upload a certificate, see How to Add an SSL Certificate. 
+   - Log Timestamp and Hostname – Set to Yes if you want to log the date and time of the event, and the hostname configured on the BASIC > IP Configuration > Domain Configuration section. 
+3. In the Logs Format section, specify values for the following fields:
+   - Syslog Header –  ArcSight Log Header 
+   - Web Firewall Logs Format –  HPE ArcSight CEF:0 
+   - Access Logs Format – HPE ArcSight CEF:0 
+   - Audit Logs Format – HPE ArcSight CEF:0 
+   - Network Firewall Logs Format - HPE ArcSight CEF:0 
+   - System Logs Format - HPE ArcSight CEF:0 
+4. Click Save.
```
Comment (ShirleyDenkberg) [inline Packs/BarracudaWAF/README.md:23]:
```suggestion
4. Click **Save**.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain @ShahafBenYakir Doc review completed.

File: Packs/BarracudaWAF/ModelingRules/BarracudaWAFModelingRules_1_3/BarracudaWAFModelingRules_1_3.yml
Comment (eepstain) [inline Packs/BarracudaWAF/ModelingRules/BarracudaWAFModelingRules_1_3/BarracudaWAFModelingRules_1_3.yml:1]: - version should be 8.2.0

- "tags:" should get commas ''

**Review by eepstain — APPROVED** [review]

**Pull request merged**